source/includes/connection-snippets/x509-snippets/php-connection.php

```diff
@@ -0,0 +1,13 @@
+<!-- Begin x509 connection -->
+<?php
+$subject = "mongodb+srv://<your_subject>"; 
+$manager = new MongoDB\Driver\Manager(
+    $subject + "@<cluster-url>/?ssl=true&authMechanism=MONGODB-X509",
+  [],
+  [
+    "pem_file" => "/etc/certs/mongodb/v3/client.pem",
+    "ca_file" => "/etc/certs/mongodb/v3/ca.pem",
+  ]
```

Comment:
Similar to my comment on the C example, this can use the `tlsCAfile` and `tlsCertificateKeyFile` URI options instead of the special driver options array being used here (per [DRIVERS-557](https://jira.mongodb.org/browse/DRIVERS-557)). That works for the 4.2-compat version of the PHP driver (extension 1.6.0+ and library 1.5.0+), and would let you only need to provide a URI string (first argument).

Similar to my other comments in the SCRAM PHP example, you don't do anything after creating the client object here so you should be aware that no IO is happening. A `ping` command would avoid that.